messagebird/client.py

```diff
@@ -88,6 +88,10 @@ def message_create(self, originator, recipients, body, params=None):
     params.update({ 'originator' : originator, 'body' : body, 'recipients' : recipients })
     return Message().load(self.request('messages', 'POST', params))
 
+  def message_delete(self, id):
+    """Delete a message from the dashboard."""
+    return Message().load(self.request('messages/' + str(id), 'DELETE'))
```

Comment:
a delete request wont return a representation of the message. So `Message().load(...)` is not useful here. Maybe you need to just check if the request is successful (status=204 No Content) and parse/present the error object otherwise.

Reply:
Thank you for pointing that out, I didn't realise that, but it makes sense. Would just using `request_plain_text`be better? Just like in `contact_delete`.

Reply:
Changed it to the same way `contact_delete` does it.